.travis.yml

```diff
@@ -17,7 +17,10 @@ cache:
   directories:
   - ~/.npm
 before_install:
+- nvm install-latest-npm
```

Comment:
Two times installing npm requires?

Reply:
probably a merge conflict, I changed `install-latest-npm` to the `npm i -g npm@6.3` line hoping to fix a build issue, but it was something different. The `npm i -g npm@6.3` line below can be removed.